### ISSUE TITLE
cache output of pbsnodes

### ISSUE THREAD

**Issue by andre-merzky** [radical-cybertools/saga-python#466]
On Archer, for example, `pbsnodes -a` takes almost a minute.  We do that over and over again.  Assuming that PBS configuration is kinda static during the lifetime of an application, this should be cached on adaptor level (per job service URL).  That is probably true for other information as well (paths to qsub etc) which all add quite a number of roundtrips for nothing, but pbsnodes is the one which takes most time.

**Comment by marksantcroos** [comment]
This has been discussed before and might be an open ticket also.

**Comment by andre-merzky** [comment]
Yeah, has been discussed, but I couldn't find a ticket.  Let me look again.

**Comment by andre-merzky** [comment]
no previous ticket found, so I'll leave this open...

**Comment by vivek-bala** [comment]
I'm not sure if this effects any other machine, but if this is confined to Archer I would suggest closing it till we get active users on Archer. @andre-merzky can you confirm we don't have users on Archer right now?

**Comment by andre-merzky** [comment]
Yeah, that affects performance on all batch adaptors.  Its 'just' an optimization though...

**Comment by vivek-bala** [comment]
I don't have the numbers, so can't judge the severity of the performance delay. Do you want to keep the ticket open since this affects all adapters?

**Comment by andre-merzky** [comment]
Yeah.  I set the priority to `low`.
